### ISSUE TITLE
Problem with my own weights

### ISSUE THREAD

**Issue by Mariquery** [guilhermesurek/computer-vision-framework#1]
Hello,
Thanks for the repository =)
I have tried it with my own weights but can't get it to work. Puts a random action and does not change despite doing other actions (Mode AR).
Any idea of ​​the problem?
Thank you very much in advance.
Greetings,
María.

**Comment by guilhermesurek** [comment]
Hi @Mariquery, good to hear that the repository helped you! Could you describe a little more the problem you are having? Which model are you using? When you say it's not working, what exactly? Is it generating an error? What are you getting and what did you expect?
Regards,
Guilherme.

**Comment by Mariquery** [comment]
Hi @guilhermesurek,
Thanks for your quick response!
Sorry, I'm new to this and I don't explain myself too well.
I have trained a resnet50 model with 6 shares with a few .pth weights.
I put the weights and names and it does not generate any error, but what I hope is that when I pass it a video it will tell me the action it performs. What I get is not the action it performs but a different one. Do I need to change anything else to make it work?
Thank you.
Regards,
María.

**Comment by guilhermesurek** [comment]
Ok, don't worry, I will try to help you. I'll ask a few questions to see if we can find out what's going on.
1- Did you train resnet50 with a set of videos?
2- How many?
3- Did you separate this set into training and validation?
4- Did you use a pre-trained model or trained from scratch?
5- How did your model perform on the training set and validation set?

**Comment by Mariquery** [comment]
Oh thank you very much!
1.- Yes, I trained resnet50 with a set of videos.
2.- 3000 approx.
3.- Yes, separated in test, training and validation.
4.- No, I trained from scratch.
5.- The precision is good.
The actions are "run" and "clap". The problem is that I am clapping and the program tells me that I am running. I think it randomly places the actions.
Thanks.

**Comment by guilhermesurek** [comment]
Ok, thank you for sharing.
In my few experiences, training from scratch with 20k vídeos always led to overfitting. Good performance in the training set (80~90%) and bad performance in the validation set (10-30%). In order of magnitude, a 3D ResNet50 has 46.4 million trainable parameters.
Could you share your accuracies for training and validation sets?
